Snakefile-sorting-hat

```diff
@@ -68,6 +68,9 @@ def get_bioentities_property_path():
 def get_delete_previous_output():
     return config['delete_previous_output']
 
+def get_skip_steps_file():
+    return config['skip_steps_file']
```

Comment:
again, consider the case when this key is not defined. Probably same applies to 'delete_previous_output' above.